synapse/handlers/sync.py

```diff
@@ -1143,8 +1133,7 @@ def _generate_sync_entry_for_device_list(
         else:
             return DeviceLists(changed=[], left=[])
 
-    @defer.inlineCallbacks
```

Comment:
Missing an await here?

Reply:
huh?

Reply:
Hm, was on mobile and didn't see an `await` before `self.store.delete_messages_for_device`. Guess that's the last time I review on mobile :)

Reply:
Ah, right, I was just confused by the fact it was on the function line. FTR `async def` doesn't need to have an `await` I don't think.